docker-compose.yml

```
@@ -0,0 +1,46 @@
+version: "3.8"
+services:
+    mongodb:
+        image: mongo
+        container_name: mongodb
+        environment:
+            - PUID=1000
+            - PGID=1000
+        volumes:
+            - db:/data/db # Necessary volume to keep database data between restarts
```

```suggestion
            - lemverse-db:/data/db # Necessary volume to keep database data between restarts
```
Maybe a prefix is better to prevent conflict for another project ? :)